pkg/ebpf/c/tracee.bpf.c

```diff
@@ -656,6 +656,306 @@ int tracepoint__sched__sched_process_fork(struct bpf_raw_tracepoint_args *ctx)
     return 0;
 }
 
+void __always_inline update_hidden_modules(u64 mod_addr, char *mod_name)
+{
+    if (bpf_map_lookup_elem(&already_reported_hidden_modules_map, &mod_addr) != NULL) {
```

Comment:
Such check can be done in userspace.
This event is not too frequent, so shouldn't be a problem to send to userspace and check there. This will allow us to remove the already_reported_hidden_modules_map